.github/workflows/wheels.yml

```
@@ -22,27 +22,33 @@ jobs:
       with:
         path: dist/*.tar.gz
 
-  #build_wheels:
-    #name: Wheel on ${{ matrix.os }}
-    #runs-on: ${{ matrix.os }}
-    #strategy:
-      #fail-fast: false
-      #matrix:
-        #os: [ubuntu-latest] #, macos-latest]
+  build_wheels:
+    name: Make ${{ matrix.python-build-version }} ${{ matrix.os }} Wheels
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-build-version: ["cp36-*", "cp37-*", "cp38-*", "cp39-*"]
+        os: [ubuntu-latest]   # , macos-latest
 
-    #steps:
-    #- uses: actions/checkout@v2
-      #with:
-        #submodules: recursive
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: Install compiler tools on macOS
+      if: runner.os == 'macOS'
+      run: brew install automake
 
-    #- uses: pypa/cibuildwheel@v2.0.0a4
-      #env:
-        #CIBW_ARCHS: auto64
+    - uses: pypa/cibuildwheel@v2.0.0a4
```

```suggestion
    - uses: pypa/cibuildwheel@v2.1.1
```

Is this still manylinux_2_24? Getting it to work on at least 2014 wouldn't likely be very helpful. 2_24 won't run on CentOS 7, for example, too old glibc.

---

I didn't set that version number.

---

When 3.3.4.0rc7 is done, I'll commit that suggestion.

---

It might not be related to this (I might be hijacking this thread), but

```
Binary wheel 'fastjet-3.3.4.0rc7-cp36-cp36m-linux_x86_64.whl' has an unsupported platform tag 'linux_x86_64'.
```

Are we missing a step from some previous dismantling? I'm going to look at the wheels.yml history.